host/libvirt_lxc_backend.go

```diff
@@ -464,6 +469,14 @@ func (l *LibvirtLXCBackend) Run(job *host.Job, runConfig *RunConfig) (err error)
 		}
 	}
 
+	for _, artifact := range job.TarArtifacts {
+		g.Log(grohl.Data{"at": "extract_tar", "uri": artifact.URI, "target_path": artifact.Attributes.TarTargetPath})
+		if err := fetchAndExtractTar(rootPath, artifact); err != nil {
```

Comment:
Let's start with it in containerinit, we can move it out if necessary for caching.